### PR TITLE
Let's make `CardSearch` better

### PR DESCRIPTION
**Pull request by MalitsPlus**
- Makes dropdown lists able to display more rows in one page.
- Puts useless skill types to the end of the dropdown list.
- Adds use original strings during searching checkbox

### CODE REVIEW

File: locales/zh-Hans/search.json [hunk 1]
```diff
@@ -21,5 +21,6 @@
     "card_filtered": "从 {total} 张卡片中找到 {num} 张卡片。",
     "st_stamina": "所需体力 {s}",
     "st_ct": "CT {ct}",
-    "st_limit": "Live 中只生效 {l} 次"
+    "st_limit": "Live 中只生效 {l} 次",
+    "Use original strings": "仅在搜索时使用日文"
```
Comment (outloudvi) [inline locales/zh-Hans/search.json:25]: * For better readability, could you move this string before all key-values whose key contains an underscore? That is to say, could you move this string before the line of `no_skip_sp_on_ctfilter`?
* Could you also add this to `en/search.json`:

``` jsonc
{
// ...,
  "Use original strings": "Use original strings",
// ...,
```

File: locales/ja/search_skills.json
Comment (outloudvi) [inline locales/ja/search_skills.json:1]: Given that you added a file at `locales/ja` that is not meant to be translated, could you add this file to `ignore_files` in `.transifex/vendor-strings.yml`? Thanks!

<https://github.com/outloudvi/info-pride/blob/bbbb5c21d2a9a7b7f8c5d8cae853e0963b2b02cc/.transifex/vendor-strings.yml#L9-L10>

**Review by outloudvi — CHANGES_REQUESTED** [review]
Thanks for your contribution! They generally seem great, but I would request for several minor edits.

**Comment by outloudvi** [comment]
@all-contributors add @MalitsPlus for code

File: locales/zh-Hans/search.json (hunk 1 above)
Comment (MalitsPlus) [reply locales/zh-Hans/search.json]: Swapped strings order.
https://github.com/outloudvi/info-pride/blob/264df9b305328e91cd764ed1b7b36deea4d7796b/locales/zh-Hans/search.json#L16-L18

Added en string.
https://github.com/outloudvi/info-pride/blob/264df9b305328e91cd764ed1b7b36deea4d7796b/locales/en/search.json#L17

**Comment by outloudvi** [comment]
Thanks for your edit! Sorry for being buried in stuffs and not being able to update it as I said before.
The build failed due to the same reason as MalitsPlus/SolisClient#5 - I'll get it merged and fix that later.

**Pull request merged**